src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertListItem.tsx

```diff
@@ -59,57 +63,40 @@ export const SavedSearchAlertListItem: React.FC<SavedSearchAlertListItemProps> =
           flex={1}
           overflow="hidden"
         >
-          <Text
-            variant={["md", "lg"]}
-            color={variant === "active" ? "blue100" : "black100"}
-            style={{ overflowWrap: "break-word" }}
-          >
-            {item.displayName}
-          </Text>
+          <Flex flexDirection="column">
+            <TextWrapper textVariant="lg-display">{item.title}</TextWrapper>
+            <TextWrapper textVariant="md">{item.subtitle}</TextWrapper>
+          </Flex>
           <Spacer x={2} y={2} />
-          <Clickable textDecoration="underline" onClick={toggleExpandFilters}>
-            <Flex flexDirection="row" alignItems="center">
-              <Text variant="sm">{toggleExpandFiltersText}</Text>
-              <Icon height={18} width={18} ml={0.5} />
-            </Flex>
-          </Clickable>
         </Flex>
         <Flex flexDirection="row" alignItems={["flex-start", "center"]}>
-          <Clickable
-            textDecoration="underline"
-            onClick={() => {
-              onEditAlertClick({
-                id: item.internalID,
-                name: item.settings?.name ?? undefined,
-                artistIds: item.artistIDs as string[],
-              })
-            }}
-          >
-            <Text variant="sm">Edit</Text>
-          </Clickable>
+          <TextWrapper textVariant="sm">
+            <Clickable
+              textDecoration="underline"
+              onClick={() => {
+                onEditAlertClick({
+                  id: item.internalID,
+                  name: item.settings?.name ?? undefined,
+                  artistIds: item.artistIDs as string[],
+                })
+              }}
+            >
+              Edit
+            </Clickable>
+          </TextWrapper>
           <Spacer x={2} />
-          <RouterLink to={viewAllHref} textDecoration="underline" zIndex={1000}>
-            <Text variant="sm">View All</Text>
-          </RouterLink>
+          <TextWrapper textVariant="sm">
+            <RouterLink
+              to={viewAllHref}
+              textDecoration="underline"
+              zIndex={1000}
```

Comment:
@dariakoko this zIndex will probably vanish after your rebase main branch, but reminding to double check it after just in case

Reply:
Yeah, thanks, I just merged with main and it's not there any more